src/common/Popup.js

```diff
@@ -14,6 +14,8 @@ const styles = createStyleSheet({
     shadowRadius: 16,
     elevation: 3,
     maxHeight: 250,
+    borderWidth: 1.5,
+    borderColor: BRAND_COLOR,
```

Comment:
Cool! This looks like it might be aimed at fixing #5018, is that right? If so, please post some [screenshots](https://zulip.readthedocs.io/en/latest/tutorials/screenshot-and-gif-software.html) in both dark mode and light mode to make it easier to see the proposed changes, and also please [put `Fixes: #5018` at the bottom of the commit message](https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#fixes-format). 🙂 Thanks!